Source/WebKit/Shared/Cocoa/WebCoreArgumentCodersCocoa.mm

```diff
@@ -253,7 +253,7 @@
     Vector<String> supportedCountries;
     if (!decoder.decode(supportedCountries))
         return false;
-    request.setSupportedCountries(WTFMove(supportedCountries));
+    request.setSupportedCountries(supportedCountries);
```

Comment:
This is an unnecessary copy.  Leave the signature of setSupportedCountries alone and call it with setSupportedCountries({ request.supportedCountries }) in convertAndValidate.

Reply:
Agreed. This way we only copy in one of the call sites. Fixed in https://github.com/WebKit/WebKit/commit/25ef05d703b8a89198b9eeea7f9dc648b8a5e71e